### PR TITLE
Added version 6.0 of kube, updated auto update for git

### PR DESCRIPTION
**Pull request by fstanis**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Downloaded version 6.0 from: https://github.com/imperavi/kube/archive/master.zip

Auto-update is still broken, as the git repo does not seem to use the correct tags ([raised as issue on their repo](https://github.com/imperavi/kube/issues/70)).

# Profile of the lib
 * Git repository (required): https://github.com/imperavi/kube
 * Official website (optional, not the repository): https://imperavi.com/kube/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 718
   - Count of watchers: 65
   - Count of forks: 139
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@fstanis Thanks. 
@pvnr0082t @naoh8103 @x09336 would you guys please take a look?

**Comment by PeterDaveHello** [comment]
The auto-update didn't work because of they didn't release new version with the git tag, we may not need this PR, the tarball should at least came from the commit for the new version like: https://github.com/imperavi/kube/archive/1b0df210415289d7fb0cd3da0b7667e5f957901f.zip.

@imperavi is it possible to add a v6.0.0 in the repo? Thanks.

**Comment by fstanis** [comment]
Hi, I feel this PR is still needed even when a new tag is added, since the directory structure on their github changed since version 3 (the basePath should now be *dist* and not empty string - I've included this change in the PR).

Also, just made a quick check, the files from this PR are identical to the files in the tarball for the commit you linked, so this should be the "official" version 6.0.

I've raised [an issue](https://github.com/imperavi/kube/issues/70) with imperavi to propose adding the correct tag to avoid this in the future, but I think a quick update right now is just as good.

**Comment by PeterDaveHello** [comment]
If the structure changed, the bot will not automatically add the version, however, I'll look into it, so that may not be a big problem :)

File: ajax/libs/kube/package.json [hunk 1]
```diff
@@ -16,10 +16,11 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/imperavi/kube.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
       "css/*",
-      "js/*"
+      "js/*",
+      "scss/*"
```
Comment (pvnr0082t) [inline ajax/libs/kube/package.json]: I think we don't need filed under `scss` folder. Let's use `"!(scss)/*"` instead of above.

Comment (fstanis) [inline ajax/libs/kube/package.json]: Fixed.

File: ajax/libs/kube/package.json [hunk 2]
```diff
@@ -16,10 +16,11 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/imperavi/kube.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
       "css/*",
-      "js/*"
+      "js/*",
+      "!(scss)/*"
```
Comment (pvnr0082t) [inline ajax/libs/kube/package.json]: Nice, and we can just keep `"!(scss)/*"` and remove `"css/*",` and `"js/*",`

Comment (fstanis) [inline ajax/libs/kube/package.json]: It seems a lot of libraries explicitly state what to include as opposed to what to excluse, so maybe it's better to just remove the scss line instead? That way, if some directory restructuring happens, we're more confident cdnjs won't pull in junk or too much data.

Comment (pvnr0082t) [inline ajax/libs/kube/package.json]: @PeterDaveHello Could you take a look? Thanks.

Comment (PeterDaveHello) [inline ajax/libs/kube/package.json]: I prefer to use "exclude" pattern, at least we can know that there are something changed, unless the author says "I can keep my eyes on it and send a PR to CDNJS when the structure changed".

**Comment by fstanis** [comment]
Sorry to bump this up, but it's been quite a while - any verdict on this?

**Comment by PeterDaveHello** [comment]
@fstanis sorry for the late reply, since GitHub won't send notification if you just update the commit in a PR without leave comment, so that our reviewer didn't notice your change, we'll do it ASAP, thanks!

**Comment by pvnr0082t** [comment]
There has conflict of this PR. Please try to resolve it. Thanks. @fstanis

**Comment by PeterDaveHello** [comment]
FYI, since v6.0.1 was added for a while:
* 0797001 add kube v6.0.1 Peter Dave Hello 3 weeks ago
* 774878a update auto-update config of kube Peter Dave Hello 3 weeks ago

You may want to close this PR and just use v6.0.1.

**Comment by fstanis** [comment]
Ah, awesome, thanks for looking into this, glad kube is now CDN-ready. :)

**Pull request closed without merging**